mofchecker/__init__.py

```diff
@@ -51,6 +51,32 @@
 __all__ = ["__version__", "MOFChecker"]
 
 
+DESCRIPTORS = [
```

Comment:
I think I wouldn't like to duplicate this. I'll try to address this when addressing the graph initialization problem you mentioned as it is related. I see that you already replaced the function that runs all check with looping over this list, but we still then have this list and the checks dictionary.

Reply:
indeed, so the duplication was already there (and I just didn't reduce it).

I'm also wondering a bit whether these different ways of calling things (overvalent, overcoordinated) are really necessary but I guess in the end it does not really hurt.